modules/developers-guide/pages/cql.adoc

```diff
@@ -0,0 +1,157 @@
+= Using the CQL API
+
```

Comment:
To use Stargate's Cassandra Query Language (CQL) API you can use the same CQL syntax, tools and native language drivers that you would with a normal Cassandra deployment. 

References for the popular Cassandra drivers are below:
* https://github.com/datastax/java-driver[Java]
* https://github.com/datastax/csharp-driver[C#]
* https://github.com/datastax/cpp-driver[C++]
* https://github.com/datastax/python-driver[Python]
* https://github.com/datastax/nodejs-driver[Node.js]
* https://github.com/gocql/gocql[Go]
* https://spring.io/projects/spring-data-cassandra[Spring Data Cassandra]

Note: there are a few key steps that you should change when using Stargate.
1. Use the addresses of the Stargate instances as contact points rather than the addresses of the Cassandra storage instances
2. Use the Round Robin policy in the driver for the https://docs.datastax.com/en/devapp/doc/devapp/driversLoadBalancing.html[Load Balancing Policy] rather than the Token Aware Policy.